src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -200,14 +200,11 @@ internal ImmutableArray<IArgumentOperation> DeriveArguments(BoundNode containing
                         var property = (PropertySymbol?)boundObjectInitializerMember.MemberSymbol;
                         Debug.Assert(property is not null);
                         return DeriveArguments(
-                                    boundObjectInitializerMember,
                                     boundObjectInitializerMember.Binder,
                                     property,
                                     boundObjectInitializerMember.Arguments,
-                                    boundObjectInitializerMember.ArgumentNamesOpt,
```

Comment:
>ArgumentNamesOpt [](start = 65, length = 16)

Is this property used anywhere else? Can it be removed?

Reply:
Yes, these and all the others are used in various places such as lowering and diagnostic passes where they are needed to visit the call.

---
In reply to: [541219591](https://github.com/dotnet/roslyn/pull/49842#discussion_r541219591) [](ancestors = 541219591)